src/accordion.test.interactions.ts

```diff
@@ -106,7 +106,7 @@ it('closes on click when animated', async () => {
     </glide-core-accordion>`,
   );
 
-  await click(host);
+  click(host);
```

Comment:
This should (🤞) resolve the [flakiness](https://github.com/CrowdStrike/glide-core/actions/runs/14040945188/job/39310726465).